core/jvm/src/main/scala/scalaz/zio/RTS.scala

```diff
@@ -52,8 +47,8 @@ trait RTS {
    * The default handler for unhandled exceptions in the main fiber, and any
    * fibers it forks that recursively inherit the handler.
    */
-  def defaultHandler: List[Throwable] => IO[Nothing, Unit] =
-    (ts: List[Throwable]) => IO.sync(ts.foreach(_.printStackTrace()))
+  def defaultHandler: Cause[Nothing] => IO[Nothing, Unit] =
+    (cause: Cause[Nothing]) => IO.sync(cause.unchecked.foreach(_.printStackTrace()))
```

Comment:
I think the type may be wrong here. If a fiber doesn't handle a checked `E`, it will end up here. So this should really be `Cause[Any]`.

Also, to print out a message, I'd use `FiberFailure(cause).message`. This way you can take advantage of the work done there to render the cause. Plus that way, you won't miss an `E` if it happens to be inside the `Cause`.